installer/create-cluster-aks.sh

```diff
@@ -45,30 +45,30 @@ function parse_args {
         CLUSTER_NAME=$1
         [ -z "$CLUSTER_NAME" ] && error '"--cluster-name" requires a non-empty option argument.\n'
```

Comment:
I think it is better to remove the "option" in these sentences. Can't we just say `--clustername requires an argument?`?

Reply:
Agreed. Will change.